module/entity-types/helper.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * name : entityTypes/helper.js
+ * author : Rakesh Kumar
+ * created-date : 01-Jun-2020
+ * Description : Entity types related helper functionality.
+ */
+
+/**
+   * EntityTypesHelper
+   * @class
+*/
+module.exports = class EntityTypesHelper {
+
+    /**
+      * List of all entity types.
+      * @method
+      * @name list
+      * @param {Object} [queryParameter = "all"] - Filtered query data.
+      * @param {Object} [projection = {}] - Projected data.   
+      * @returns {Object} returns a entity types list from the filtered data.
+     */
+
+    static list(queryParameter = "all", projection = {}) {
```

Comment:
@rakeshSgr - Projection can be an array, which controller can pass what all fields they want. Check solutionDocuments  function here -  https://github.com/shikshalokam/sl-assessments-service/blob/development/module/solutions/helper.js

Reply:
resolved